### PR TITLE
updating social sharing copy

### PR DESCRIPTION
**Pull request by cbradshaw**
disregard the "javascript_validation" in the title. This just ended up being changes to the social sharing copy.

### CODE REVIEW

**Comment by RanSolo** [comment]
:+1: looks like a pretty straight forward text change.

**Comment by RanSolo** [comment]
has conflicts though I see

**Comment by jchapin** [comment]
Large conflicting swath of content on ```app/views/user_mailer/signup_email.html.erb```

```html
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
    <title>The Mindfulness Edge</title>
    <style type="text/css">
      /* /\/\/\/\/\/\/\/\/ CLIENT-SPECIFIC STYLES /\/\/\/\/\/\/\/\/ */
      #outlook a{padding:0;} /* Force Outlook to provide a "view in browser" message */
      .ReadMsgBody{width:100%;} .ExternalClass{width:100%;} /* Force Hotmail to display emails at full width */
      .ExternalClass, .ExternalClass p, .ExternalClass span, .ExternalClass font, .ExternalClass td, .ExternalClass div {line-height: 100%;} /* Force Hotmail to display normal line spacing */
      body, table, td, p, a, li, blockquote{-webkit-text-size-adjust:100%; -ms-text-size-adjust:100%;} /* Prevent WebKit and Windows mobile changing default text sizes */
      table, td{mso-table-lspace:0pt; mso-table-rspace:0pt;} /* Remove spacing between tables in Outlook 2007 and up */
      img{-ms-interpolation-mode:bicubic;} /* Allow smoother rendering of resized image in Internet Explorer */

      /* /\/\/\/\/\/\/\/\/ RESET STYLES /\/\/\/\/\/\/\/\/ */
      body{margin:0; padding:0;}
      img{border:0; height:auto; line-height:100%; outline:none; text-decoration:none;}
      table{border-collapse:collapse !important;}
      body, #bodyTable, #bodyCell{height:100% !important; margin:0; padding:0; width:100% !important; font-family: Helvetica, Arial}

      /* /\/\/\/\/\/\/\/\/ TEMPLATE STYLES /\/\/\/\/\/\/\/\/ */

      /* ========== Page Styles ========== */

      #bodyCell{padding:20px;}
      #templateContainer{width:600px;}

      /**
      * @tab Page
      * @section background style
      * @tip Set the background color and top border for your email. You may want to choose colors that match your company's branding.
      * @theme page
      */
      body, #bodyTable{
        background-color:#EEE;
      }

      /**
      * @tab Page
      * @section background style
      * @tip Set the background color and top border for your email. You may want to choose colors that match your company's branding.
      * @theme page
      */
      #bodyCell{
      }

      /**
      * @tab Page
      * @section email border
      * @tip Set the border for your email.
      */
      #templateContainer{
        border:1px solid #BBBBBB;
      }

      /**
      * @tab Page
      * @section heading 1
      * @tip Set the styling for all first-level headings in your emails. These should be the largest of your headings.
      * @style heading 1
      */
      h1{
        color:#404040 !important;
        display:block;
        font-family:Georgia;
        font-size:26px;
        font-style:normal;
        font-weight:bold;
        line-height:100%;
        letter-spacing:normal;
        margin-top:0;
        margin-right:0;
        margin-bottom:10px;
        margin-left:0;
        text-align:left;
      }

      /**
      * @tab Page
      * @section heading 2
      * @tip Set the styling for all second-level headings in your emails.
      * @style heading 2
      */
      h2{
        color:#404040 !important;
        display:block;
        font-family:Helvetica;
        font-size:20px;
        font-style:normal;
        font-weight:bold;
        line-height:100%;
        letter-spacing:normal;
        margin-top:0;
        margin-right:0;
        margin-bottom:10px;
        margin-left:0;
        text-align:left;
      }

      /**
      * @tab Page
      * @section heading 3
      * @tip Set the styling for all third-level headings in your emails.
      * @style heading 3
      */
      h3{
        color:#404040 !important;
        display:block;
        font-family:Helvetica;
        font-size:16px;
        font-style:normal;
        font-weight:bold;
        line-height:100%;
        letter-spacing:normal;
        margin-top:0;
        margin-right:0;
        margin-bottom:10px;
        margin-left:0;
        text-align:left;
      }

      /**
      * @tab Page
      * @section heading 4
      * @tip Set the styling for all fourth-level headings in your emails. These should be the smallest of your headings.
      * @style heading 4
      */
      h4{
    color: #404040 !important;
    font-family: Helvetica;
    font-size: 14px;
    font-style: normal;
    line-height: 125%;
    letter-spacing: normal;
    margin: 0px 0px 10px;
    text-align: left;
    font-weight: normal !important;
    padding: 0px 15px !important;
    display: block !important;
      }

      /* ========== Header Styles ========== */

      /**
      * @tab Header
      * @section preheader style
      * @tip Set the background color and bottom border for your email's preheader area.
      * @theme header
      */
      #templatePreheader{
        border-bottom:1px solid #CCCCCC;
      }

      /**
      * @tab Header
      * @section preheader text
      * @tip Set the styling for your email's preheader text. Choose a size and color that is easy to read.
      */
      .preheaderContent{
        color:#808080;
        font-family:Helvetica;
        font-size:10px;
        line-height:125%;
        text-align:left;
      }

      /**
      * @tab Header
      * @section preheader link
      * @tip Set the styling for your email's preheader links. Choose a color that helps them stand out from your text.
      */
      .preheaderContent a:link, .preheaderContent a:visited, /* Yahoo! Mail Override */ .preheaderContent a .yshortcuts /* Yahoo! Mail Override */{
        color:#606060;
        font-weight:normal;
        text-decoration:underline;
      }


      /**
      * @tab Header
      * @section header text
      * @tip Set the styling for your email's header text. Choose a size and color that is easy to read.
      */
      .headerContent{
        color:#505050;
        font-family:Helvetica;
        font-size:20px;
        font-weight:bold;
        line-height:100%;
        padding-top:0;
        padding-right:0;
        padding-bottom:0;
        padding-left:0;
        text-align:left;
        vertical-align:middle;
      }

      /**
      * @tab Header
      * @section header link
      * @tip Set the styling for your email's header links. Choose a color that helps them stand out from your text.
      */
      .headerContent a:link, .headerContent a:visited, /* Yahoo! Mail Override */ .headerContent a .yshortcuts /* Yahoo! Mail Override */{
        color:#EB4102;
        font-weight:normal;
        text-decoration:underline;
      }
<<<<<<< HEAD
      
			/* ========== Footer Styles ========== */

			/**
			* @tab Footer
			* @section footer style
			* @tip Set the background color and borders for your email's footer area.
			* @theme footer
			*/
			#templateFooter{
				background-color:#FFFFFF;
				border-top: #DDD 1px solid;
			}

			/**
			* @tab Footer
			* @section footer text
			* @tip Set the styling for your email's footer text. Choose a size and color that is easy to read.
			* @theme footer
			*/
			.footerContent{
				color:#808080;
				font-family:Helvetica;
				font-size:10px;
				line-height:150%;
				padding-top:20px;
				padding-right:20px;
				padding-bottom:20px;
				padding-left:20px;
				text-align:center;
			}
			

			/**
			* @tab Footer
			* @section footer link
			* @tip Set the styling for your email's footer links. Choose a color that helps them stand out from your text.
			*/
			.footerContent a:link, .footerContent a:visited, /* Yahoo! Mail Override */ .footerContent a .yshortcuts, .footerContent a span /* Yahoo! Mail Override */{
			    color: #040404;
			    font-weight: normal;
			    text-decoration: none;
			    font-size: 14px;
			}
			/* /\/\/\/\/\/\/\/\/ MOBILE STYLES /\/\/\/\/\/\/\/\/ */

            @media only screen and (max-width: 480px){
				/* /\/\/\/\/\/\/ CLIENT-SPECIFIC MOBILE STYLES /\/\/\/\/\/\/ */
				body, table, td, p, a, li, blockquote{-webkit-text-size-adjust:none !important;} /* Prevent Webkit platforms from changing default text sizes */
                body{width:100% !important; min-width:100% !important;} /* Prevent iOS Mail from adding padding to the body */

				/* /\/\/\/\/\/\/ MOBILE RESET STYLES /\/\/\/\/\/\/ */
				#bodyCell{padding:10px !important;}

				/* /\/\/\/\/\/\/ MOBILE TEMPLATE STYLES /\/\/\/\/\/\/ */

				/* ======== Page Styles ======== */

				/**
				* @tab Mobile Styles
				* @section template width
				* @tip Make the template fluid for portrait or landscape view adaptability. If a fluid layout doesn't work for you, set the width to 300px instead.
				*/
				#templateContainer{
					max-width:600px !important;
					width:100% !important;
				}

				/**
				* @tab Mobile Styles
				* @section heading 1
				* @tip Make the first-level headings larger in size for better readability on small screens.
				*/
				h1{
					font-size:24px !important;
					line-height:100% !important;
				}

				/**
				* @tab Mobile Styles
				* @section heading 2
				* @tip Make the second-level headings larger in size for better readability on small screens.
				*/
				h2{
					font-size:20px !important;
					line-height:100% !important;
				}

				/**
				* @tab Mobile Styles
				* @section heading 3
				* @tip Make the third-level headings larger in size for better readability on small screens.
				*/
				h3{
					font-size:18px !important;
					line-height:100% !important;
				}

				/**
				* @tab Mobile Styles
				* @section heading 4
				* @tip Make the fourth-level headings larger in size for better readability on small screens.
				*/
				h4{
					font-size:16px !important;
					line-height:100% !important;
					font-weight: bold!important;
					font-family: Helvetica!important;
					border-left: #D5DE22 4px solid!important;
					padding: 0 0 0 15px!important;
					display: block!important;
					font-style: normal;
				}

				.callout {
					font-weight: bold;
					font-family: Helvetica;
					border-left: 4px solid #D5DE22;
					padding: 0 0 0 15px;
				}


				/* ======== Header Styles ======== */

				#templatePreheader{display:none !important;} /* Hide the template preheader to save space */

				/**
				* @tab Mobile Styles
				* @section header image
				* @tip Make the main header image fluid for portrait or landscape view adaptability, and set the image's original width as the max-width. If a fluid setting doesn't work, set the image width to half its original size instead.
				*/
				#headerImage{
					height:auto !important;
					max-width:600px !important;
					width:100% !important;
				}

				/**
				* @tab Mobile Styles
				* @section header text
				* @tip Make the header content text larger in size for better readability on small screens. We recommend a font size of at least 16px.
				*/
				.headerContent{
					font-size:20px !important;
					line-height:125% !important;
				}

				/* ======== Body Styles ======== */

				/**
				* @tab Mobile Styles
				* @section body text
				* @tip Make the body content text larger in size for better readability on small screens. We recommend a font size of at least 16px.
				*/
				.bodyContent{
					font-size:18px !important;
					line-height:125% !important;
				}

				/* ======== Column Styles ======== */

				.templateColumnContainer{display:block !important; width:100% !important;}

				/**
				* @tab Mobile Styles
				* @section column image
				* @tip Make the column image fluid for portrait or landscape view adaptability, and set the image's original width as the max-width. If a fluid setting doesn't work, set the image width to half its original size instead.
				*/
				.columnImage{
					height:auto !important;
					max-width:480px !important;
					width:100% !important;
				}

				/**
				* @tab Mobile Styles
				* @section left column text
				* @tip Make the left column content text larger in size for better readability on small screens. We recommend a font size of at least 16px.
				*/
				.leftColumnContent{
					font-size:16px !important;
					line-height:125% !important;
				}

				/**
				* @tab Mobile Styles
				* @section right column text
				* @tip Make the right column content text larger in size for better readability on small screens. We recommend a font size of at least 16px.
				*/
				.rightColumnContent{
					font-size:16px !important;
					line-height:125% !important;
				}

				/* ======== Footer Styles ======== */

				/**
				* @tab Mobile Styles
				* @section footer text
				* @tip Make the body content text larger in size for better readability on small screens.
				*/
				.footerContent{
					font-size:14px !important;
					line-height:115% !important;
				}

				.footerContent a{display:block !important;} /* Place footer social and utility links on their own lines, for easier access */
			}
			
			
		</style>
    </head>
    <body leftmargin="0" marginwidth="0" topmargin="0" marginheight="0" offset="0">
    	<center>
        	<table align="center" border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" id="bodyTable">
            	<tr>
                	<td align="center" valign="top" id="bodyCell">
                    	<!-- BEGIN TEMPLATE // -->
                    	<table border="0" cellpadding="0" cellspacing="0" id="templateContainer">
                        	<tr>
                            	<td align="center" valign="top">
                                	<!-- BEGIN BODY // -->
                                    <table border="0" cellpadding="0" cellspacing="0" width="100%" id="templateBody">
                                        <tr>
                                            <td valign="top" class="bodyContent">
                                                <h1 class="page_title"><%= @user.first_name %>, thanks for buying the book!</h1>
                                                <h2>You should receive your rewards via e-mail by March 7th.</h2>
                                            </td>
                                        </tr>
                                    </table>
                                    <!-- // END BODY -->
                                </td>
                            </tr>
                        	<tr>
                            	<td align="center" valign="top">
                                	<!-- BEGIN COLUMNS // -->
                                    <table border="0" cellpadding="0" cellspacing="0" width="100%" id="templateColumns">
                                    	<tr>
                                        	<td align="center" valign="top" class="templateColumnContainer" style="padding-top:20px;">
                                            	<table border="0" cellpadding="20" cellspacing="0" width="100%">
                                                    <tr>
                                                    	<td valign="top" class="leftColumnContent centered">
                                                        <%= image_tag("tenney_mindfulness_edge.jpg", class: 'book_img') %>
                                                      </td>
                                                    </tr>
                                                </table>
                                            </td>
                                        	<td align="center" valign="top" class="templateColumnContainer" style="padding-top:20px;">
                                            	<table border="0" cellpadding="20" cellspacing="0" width="100%">
                                                    <tr>
                                                    	<td valign="top" class="rightColumnContent share_content">
														
                                                        <h4 class="why" style="text-align: center;">Don't leave your friends out</h4>
                                                        <h2 class="title" style="text-align: center;">Invite Friends &amp;<br>Earn Rewards</h2>
                                                        <h3 class="subtitle" style="text-align: center;">Share your unique link below via email, Facebook or Twitter to earn the rewards you see below.</h3>
                                                        <div class="copy-link" style="text-align: center;"><%= root_url %>?ref=<%= @user.referral_code %></div>
                                              
                                                        <% twitter_message = "I can't wait to read the new book #TheMindfulnessEdge! And, I just got $308 in giveaways for ordering it!" %>
  
                                                        <div class="social-links">
                                                          <a href="http://www.facebook.com/sharer/sharer.php?u=<%= CGI::escape(root_url) %>?ref=<%= CGI::escape(@user.referral_code) %>&title=The Mindfulness Edge" class="fb" target="_blank"><%= image_tag("fb.png", class: 'fb_img') %></a>
                                                          <div class="sep"></div>
                                                          <a href="http://twitter.com/share?url=<%= CGI::escape(root_url) %>?ref=<%= CGI::escape(@user.referral_code) %>&text=<%= CGI::escape(twitter_message) %>" class="twit" target="_blank"><%= image_tag("twit.png", class: 'tweet_img') %></a>
                                                        </div>

                                                      </td>
                                                    </tr>
                                                </table>
                                            </td>
                                        </tr>
                                    </table>
                                    <!-- // END COLUMNS -->
                                </td>
=======

      #headerImage{
        height:auto;
        max-width:600px;
      }

      /* ========== Body Styles ========== */

    .centered {
    text-align: center;
    }

      /**
      * @tab Body
      * @section body style
      * @tip Set the background color and borders for your email's body area.
      */
      #templateBody{
        background-color:#FFFFFF;
      }

      /**
      * @tab Body
      * @section body text
      * @tip Set the styling for your email's main content text. Choose a size and color that is easy to read.
      * @theme main
      */
      .bodyContent{
        color:#505050;
        font-family:Helvetica;
        font-size:16px;
        line-height:150%;
        padding-top:20px;
        padding-right:20px;
        padding-bottom:20px;
        padding-left:20px;
        text-align:left;
      }

      /**
      * @tab Body
      * @section body link
      * @tip Set the styling for your email's main content links. Choose a color that helps them stand out from your text.
      */
      .bodyContent a:link, .bodyContent a:visited, /* Yahoo! Mail Override */ .bodyContent a .yshortcuts /* Yahoo! Mail Override */{
        color:#5BB9DC;
        font-weight:normal;
        text-decoration:underline;
      }

      .bodyContent a:hover, .bodyContent a:active, /* Yahoo! Mail Override */ .bodyContent a .yshortcuts /* Yahoo! Mail Override */{
        color:#FFC001;
        font-weight:normal;
        text-decoration:underline;
      }

      .bodyContent img{
        display:inline;
        height:auto;
        max-width:560px;
      }

      /* ========== Column Styles ========== */

      .templateColumnContainer{width:260px;}

      /**
      * @tab Columns
      * @section column style
      * @tip Set the background color and borders for your email's column area.
      */
      #templateColumns{
        background-color:#FFFFFF;
      }

      /**
      * @tab Columns
      * @section left column text
      * @tip Set the styling for your email's left column content text. Choose a size and color that is easy to read.
      */
      .leftColumnContent{
        color:#505050;
        font-family:Helvetica;
        font-size:14px;
        line-height:150%;
        padding-top:0;
        padding-right:20px;
        padding-bottom:20px;
        padding-left:20px;
        text-align:left;
      }

      /**
      * @tab Columns
      * @section left column link
      * @tip Set the styling for your email's left column content links. Choose a color that helps them stand out from your text.
      */
      .leftColumnContent a:link, .leftColumnContent a:visited, /* Yahoo! Mail Override */ .leftColumnContent a .yshortcuts /* Yahoo! Mail Override */{
        color:#EB4102;
        font-weight:normal;
        text-decoration:underline;
      }

      /**
      * @tab Columns
      * @section right column text
      * @tip Set the styling for your email's right column content text. Choose a size and color that is easy to read.
      */
      .rightColumnContent{
        color:#505050;
        font-family:Helvetica;
        font-size:14px;
        line-height:150%;
        padding-top:0;
        padding-right:20px;
        padding-bottom:20px;
        padding-left:20px;
        text-align:left;
      }

      /**
      * @tab Columns
      * @section right column link
      * @tip Set the styling for your email's right column content links. Choose a color that helps them stand out from your text.
      */
      .rightColumnContent a:link, .rightColumnContent a:visited, /* Yahoo! Mail Override */ .rightColumnContent a .yshortcuts /* Yahoo! Mail Override */{
        color:#EB4102;
        font-weight:normal;
        text-decoration:underline;
      }

      .leftColumnContent img, .rightColumnContent img{
        display:inline;
        height:auto;
        max-width:260px;
      }

    .friends_cell, .rewards_cell {
    border-bottom: #DDD 1px solid;
    }

    .copy-link {
      border: 1px solid #D1D0C7;
      background: #F1F1F1 none repeat scroll 0% 0%;
      padding: 10px 20px;
      width: auto;
      text-transform: none;
      font-size: 12px;
      margin: 5px auto 0px;
      font-weight: 500;
      letter-spacing: 0.1em;
      display: inline-block;
      box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.25) inset;
    }

    .book_img {
    display: block;
    margin: 0 auto;
    }

      /* ========== Footer Styles ========== */

      /**
      * @tab Footer
      * @section footer style
      * @tip Set the background color and borders for your email's footer area.
      * @theme footer
      */
      #templateFooter{
        background-color:#FFFFFF;
        border-top: #DDD 1px solid;
      }

      /**
      * @tab Footer
      * @section footer text
      * @tip Set the styling for your email's footer text. Choose a size and color that is easy to read.
      * @theme footer
      */
      .footerContent{
        color:#808080;
        font-family:Helvetica;
        font-size:10px;
        line-height:150%;
        padding-top:20px;
        padding-right:20px;
        padding-bottom:20px;
        padding-left:20px;
        text-align:center;
      }


      /**
      * @tab Footer
      * @section footer link
      * @tip Set the styling for your email's footer links. Choose a color that helps them stand out from your text.
      */
      .footerContent a:link, .footerContent a:visited, /* Yahoo! Mail Override */ .footerContent a .yshortcuts, .footerContent a span /* Yahoo! Mail Override */{
        color: #040404;
        font-weight: normal;
        text-decoration: none;
        font-size: 14px;
      }
      /* /\/\/\/\/\/\/\/\/ MOBILE STYLES /\/\/\/\/\/\/\/\/ */

      @media only screen and (max-width: 480px){
        /* /\/\/\/\/\/\/ CLIENT-SPECIFIC MOBILE STYLES /\/\/\/\/\/\/ */
        body, table, td, p, a, li, blockquote{-webkit-text-size-adjust:none !important;} /* Prevent Webkit platforms from changing default text sizes */
        body{width:100% !important; min-width:100% !important;} /* Prevent iOS Mail from adding padding to the body */

        /* /\/\/\/\/\/\/ MOBILE RESET STYLES /\/\/\/\/\/\/ */
        #bodyCell{padding:10px !important;}

        /* /\/\/\/\/\/\/ MOBILE TEMPLATE STYLES /\/\/\/\/\/\/ */

        /* ======== Page Styles ======== */

        /**
        * @tab Mobile Styles
        * @section template width
        * @tip Make the template fluid for portrait or landscape view adaptability. If a fluid layout doesn't work for you, set the width to 300px instead.
        */
        #templateContainer{
          max-width:600px !important;
          width:100% !important;
        }

        /**
        * @tab Mobile Styles
        * @section heading 1
        * @tip Make the first-level headings larger in size for better readability on small screens.
        */
        h1{
          font-size:24px !important;
          line-height:100% !important;
        }

        /**
        * @tab Mobile Styles
        * @section heading 2
        * @tip Make the second-level headings larger in size for better readability on small screens.
        */
        h2{
          font-size:20px !important;
          line-height:100% !important;
        }

        /**
        * @tab Mobile Styles
        * @section heading 3
        * @tip Make the third-level headings larger in size for better readability on small screens.
        */
        h3{
          font-size:18px !important;
          line-height:100% !important;
        }

        /**
        * @tab Mobile Styles
        * @section heading 4
        * @tip Make the fourth-level headings larger in size for better readability on small screens.
        */
        h4{
          font-size:16px !important;
          line-height:100% !important;
          font-weight: bold!important;
          font-family: Helvetica!important;
          border-left: #D5DE22 4px solid!important;
          padding: 0 0 0 15px!important;
          display: block!important;
          font-style: normal;
        }

        .callout {
          font-weight: bold;
          font-family: Helvetica;
          border-left: 4px solid #D5DE22;
          padding: 0 0 0 15px;
        }


        /* ======== Header Styles ======== */

        #templatePreheader{display:none !important;} /* Hide the template preheader to save space */

        /**
        * @tab Mobile Styles
        * @section header image
        * @tip Make the main header image fluid for portrait or landscape view adaptability, and set the image's original width as the max-width. If a fluid setting doesn't work, set the image width to half its original size instead.
        */
        #headerImage{
          height:auto !important;
          max-width:600px !important;
          width:100% !important;
        }

        /**
        * @tab Mobile Styles
        * @section header text
        * @tip Make the header content text larger in size for better readability on small screens. We recommend a font size of at least 16px.
        */
        .headerContent{
          font-size:20px !important;
          line-height:125% !important;
        }

        /* ======== Body Styles ======== */

        /**
        * @tab Mobile Styles
        * @section body text
        * @tip Make the body content text larger in size for better readability on small screens. We recommend a font size of at least 16px.
        */
        .bodyContent{
          font-size:18px !important;
          line-height:125% !important;
        }

        /* ======== Column Styles ======== */

        .templateColumnContainer{display:block !important; width:100% !important;}

        /**
        * @tab Mobile Styles
        * @section column image
        * @tip Make the column image fluid for portrait or landscape view adaptability, and set the image's original width as the max-width. If a fluid setting doesn't work, set the image width to half its original size instead.
        */
        .columnImage{
          height:auto !important;
          max-width:480px !important;
          width:100% !important;
        }

        /**
        * @tab Mobile Styles
        * @section left column text
        * @tip Make the left column content text larger in size for better readability on small screens. We recommend a font size of at least 16px.
        */
        .leftColumnContent{
          font-size:16px !important;
          line-height:125% !important;
        }

        /**
        * @tab Mobile Styles
        * @section right column text
        * @tip Make the right column content text larger in size for better readability on small screens. We recommend a font size of at least 16px.
        */
        .rightColumnContent{
          font-size:16px !important;
          line-height:125% !important;
        }

        /* ======== Footer Styles ======== */

        /**
        * @tab Mobile Styles
        * @section footer text
        * @tip Make the body content text larger in size for better readability on small screens.
        */
        .footerContent{
          font-size:14px !important;
          line-height:115% !important;
        }

        .footerContent a{display:block !important;} /* Place footer social and utility links on their own lines, for easier access */
      }


    </style>
  </head>
  <body leftmargin="0" marginwidth="0" topmargin="0" marginheight="0" offset="0">
    <center>
      <table align="center" border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" id="bodyTable">
        <tr>
          <td align="center" valign="top" id="bodyCell">
            <!-- BEGIN TEMPLATE // -->
            <table border="0" cellpadding="0" cellspacing="0" id="templateContainer">
              <tr>
                <td align="center" valign="top">
                  <!-- BEGIN BODY // -->
                  <table border="0" cellpadding="0" cellspacing="0" width="100%" id="templateBody">
                    <tr>
                      <td valign="top" class="bodyContent">
                        <h1 class="page_title"><%= @user.first_name %>, thanks for buying the book!</h1>
                      </td>
                    </tr>
                  </table>
                  <!-- // END BODY -->
                </td>
              </tr>
              <tr>
                <td align="center" valign="top">
                  <!-- BEGIN COLUMNS // -->
                  <table border="0" cellpadding="0" cellspacing="0" width="100%" id="templateColumns">
                    <tr>
                      <td align="center" valign="top" class="templateColumnContainer" style="padding-top:20px;">
                        <table border="0" cellpadding="20" cellspacing="0" width="100%">
                          <tr>
                            <td valign="top" class="leftColumnContent centered">
                            <%= image_tag("tenney_mindfulness_edge.jpg", class: 'book_img') %>
                            </td>
                          </tr>
                        </table>
                      </td>
                      <td align="center" valign="top" class="templateColumnContainer" style="padding-top:20px;">
                        <table border="0" cellpadding="20" cellspacing="0" width="100%">
                          <tr>
                            <td valign="top" class="rightColumnContent share_content">

                            <h4 class="why" style="text-align: center;">Don't leave your friends out</h4>
                            <h2 class="title" style="text-align: center;">Invite Friends &amp;<br>Earn Rewards</h2>
                            <h3 class="subtitle" style="text-align: center;">Share your unique link below via email, Facebook or Twitter to earn the rewards you see below.</h3>
                            <div class="copy-link" style="text-align: center;"><%= root_url %>?ref=<%= @user.referral_code %></div>

                            <% twitter_message = "Check out The Mindfulness Edge book!" %>

                            <div class="social-links">
                              <a href="http://www.facebook.com/sharer/sharer.php?u=<%= CGI::escape(root_url) %>?ref=<%= CGI::escape(@user.referral_code) %>&title=The Mindfulness Edge" class="fb" target="_blank"><%= image_tag("fb.png", class: 'fb_img') %></a>
                              <div class="sep"></div>
                              <a href="http://twitter.com/share?url=<%= CGI::escape(root_url) %>?ref=<%= CGI::escape(@user.referral_code) %>&text=<%= CGI::escape(twitter_message) %>" class="twit" target="_blank"><%= image_tag("twit.png", class: 'tweet_img') %></a>
                            </div>

                            </td>
                          </tr>
                        </table>
                      </td>
                    </tr>
                  </table>
                  <!-- // END COLUMNS -->
                </td>
              </tr>


              <tr>
                <td align="center" valign="top">
                  <!-- BEGIN BODY // -->
                  <table border="0" cellpadding="0" cellspacing="0" width="100%" id="templateBody">
                    <tr>
                      <td valign="top" class="bodyContent">

                          <table id="rewards_table" class="detailed_rewards" cellpadding="5" cellspacing="5">
                            <tr>
                              <th class="friends_cell">Item</th>
                              <th class="rewards_cell">Value</th>
                            </tr>
                            <tr valign="top">
                              <td class="friends_cell">1. The complete <a href="http://jongordon.com/createacultureofgreatness.html" target="_blank">Creating a Culture of Greatness</a> program by bestselling author of The Energy Bus, Jon Gordon.</td>
                              <td class="rewards_cell">$99</td>
>>>>>>> master
                            </tr>
                            <tr valign="top">
                              <td class="friends_cell">2. The Your Brain Matters series.  A collection of brain-friendly, bite-sized videos on optimizing your brain for success from <a href="http://completeintelligence.com/your-brain-matters-video-series/" target="_blank">Scott Halford</a>, author of Activate Your Brain.</td>
                              <td class="rewards_cell">$97</td>
                            </tr>
                            <tr valign="top">
                              <td class="friends_cell">3. A 1-hour Mindfulness Edge webinar with authors <a href="http://www.themindfulnessedge.com/media.html" target="_blank">Tim Gard and Matt Tenney</a> (3 time slots will be available)</td>
                              <td class="rewards_cell">$50</td>
                            </tr>
                            <tr valign="top">
                              <td class="friends_cell">4. 25% off an annual subscription to the productivity software, <a href="https://nozbe.com/" target="_blank"><em>Nozbe</em></a>.</td>
                              <td class="rewards_cell">$24</td>
                            </tr>
                            <tr valign="top">
                              <td class="friends_cell">5. A two-week membership to the <a href="http://getmomentum.co/get-momentum/" target="_blank">Get Momentum Coaching Program</a> by Jason Womack, productivity expert and author of Get Momentum.</td>
                              <td class="rewards_cell">$20</td>
                            </tr>
                            <tr valign="top">
                              <td class="friends_cell">6. A free copy of the book <a href="http://johnspence.com/books/letters-to-a-ceo/" target="_blank">Letters to a CEO</a> (as well as some priceless videos and articles) by John Spence, named one of the top 500 leadership development experts in the world.</td>
                              <td class="rewards_cell">$10</td>
                            </tr>
                            <tr valign="top">
                              <td class="friends_cell">7. A free copy of the book <a href="http://gordontredgold.com/my-book/" target="_blank">A Practitioners Guide to Leadership</a> by Gordon Tregold, named by Inc. Magazine as one of the top 100 Leadership and Management Experts and Speakers.</td>
                              <td class="rewards_cell">$8.56</td>
                            </tr>
                          </table>

                      </td>
                    </tr>
                  </table>
                  <!-- // END BODY -->
                </td>
              </tr>
              <tr>
                <td align="center" valign="top">
                  <!-- BEGIN FOOTER // -->
                  <table border="0" cellpadding="0" cellspacing="0" width="100%" id="templateFooter">
                    <tr>
                      <td valign="top" class="footerContent" style="text-align: center;">
                        <a href="http://themindfulnessedge.com/">www.themindfulnessedge.com</a>
                      </td>
                    </tr>
                  </table>
                  <!-- // END FOOTER -->
                </td>
              </tr>
            </table>
            <!-- // END TEMPLATE -->
          </td>
        </tr>
      </table>
    </center>
  </body>
</html>
```

**Comment by jchapin** [comment]
The test changes from the commits seem to be preserved. The changes otherwise are differences in indentation settings in our various editors.

File: app/views/user_mailer/signup_email.html.erb [hunk 1]
```diff
@@ -206,165 +206,6 @@
...
-      letter-spacing: 0.1em;
-      display: inline-block;
-      box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.25) inset;
-    }
-
-    .book_img {
-    display: block;
-    margin: 0 auto;
-    }
-
```
Comment (jchapin) [inline app/views/user_mailer/signup_email.html.erb]: A large block of styling is being pulled out.

**Comment by jchapin** [comment]
Pushing this to dev and testing the e-mail before merging this.

**Comment by jchapin** [comment]
This diverged too far from master. We're better off starting a new branch with the changes applied.

**Comment by jchapin** [comment]
I can't find a PR with the original open graph tag implementations at all. This may be a follow on to a PR that was never merged in the first place.

**Comment by jchapin** [comment]
This PR attached to the original Harry's project has the OpenGraph content.

https://github.com/harrystech/prelaunchr/pull/36

That one is going to need to be shut down. We'll have to take the template from this branch whole.

**Comment by jchapin** [comment]
The diff looks better, not a large block of styling being pulled out.

**Comment by jchapin** [comment]
:+1:

**Pull request merged**